### ISSUE TITLE
Very confused

### ISSUE THREAD

**Issue by bradymiller** [openemr/blog#7]
I am really confused here. The master branch only has 4 articles, but the site is up to 7 ??
Am I missing something?
@robertdown and @MatthewVita

**Comment by bradymiller** [comment]
Here is a diff of index.html of what is on the production blog vs what is being produced by master branch hugo:
https://github.com/openemr/blog/pull/8/commits/104ab26662601064529f4d33050201b659903dca?w=1

There are 3 missing entries. And I can't find them on this repo. Is there missing work on private repo(s) not yet in the this public repo?

**Comment by bradymiller** [comment]
@MatthewVita and @robertdown ,
Just got the OpenEMR, A Great Long Term Choice blog entry up. Note I had to go through the work around to get this working. Regarding the 3 missing blog entries, just send me the .md files and the pics and I'll gladly get it going (it will then be much easier to update the blog). Note I did some serious spring cleaning of this repo (it is now running with bare bones and even removed tags/categories stuff since we were not using it; can easily add this back if plan to in the future).
btw, the 3 missing blog entries that I need are:
Embracing Docker
OpenEMR at VCU HealthHacks
OpenEMR at WONCA SAR
